psh/echo/echo.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Phoenix-RTOS
+ *
+ * echo - display a line of text
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "../psh.h"
+
+
+void psh_echoinfo(void)
+{
+	printf("display a line of text");
+}
+
+
+static void psh_echo_help(const char *prog)
+{
+	printf("Usage: %s [options] [string]\n", prog);
+	printf("  -h:  shows this help message\n");
+}
+
+
+size_t psh_echoPrintVar(const char *var)
```

Comment:
I'm not sure about function naming convention. In other applets I have seen at least 2 conventions, - `psh_echo_printVar` or simply: `echo_printVar`. The question is, which should we follow and do we want a third one? The same with a previous function, it could be `echo_help()` as well.

Reply:
Also:
```suggestion
static size_t psh_echoPrintVar(const char *var)
```

Reply:
Yeah, it's a mess. I don't see a point of psh_ prefix, but I based my echo on cat, so I borrowed it's style. I changed this function name to adhere to this style (was new style altogether).